diesel_tests/tests/expressions/mod.rs

```diff
@@ -48,7 +48,27 @@ table! {
 }
 
 #[test]
-fn test_count_max() {
+fn test_min() {
+    use self::numbers::columns::*;
```

Comment:
`use self::numbers::dsl::*;`

Reply:
This is the test that I just moved up from below, which is why I didn't change it when you said to use the api for new tests.
But I'll go ahead and migrate to diesel public api for both the test_min and test_max functions since they're related.